src/nucleus/tt.ml

```diff
@@ -6,6 +6,8 @@ type ('a,'b) constrain =
 
 type ('a, 'b) abstraction = (Name.ident * 'a) * 'b
 
+type bound = int
```

Comment:
The explanation is that using `Syntax.bound` woudl create a dependency cycle. We will deal with this in #270.